plugins/extractors/http/http_extractor_test.go

```diff
@@ -890,4 +1007,6 @@ func replaceServerURL(cfg map[string]interface{}, serverURL string) {
 	reqCfg["url"] = strings.Replace(reqCfg["url"].(string), "{{serverURL}}", serverURL, 1)
 	scriptCfg := cfg["script"].(map[string]interface{})
 	scriptCfg["source"] = strings.Replace(scriptCfg["source"].(string), "{{serverURL}}", serverURL, -1)
+	beforeScriptCfg := cfg["before_script"].(map[string]interface{})
+	beforeScriptCfg["source"] = strings.Replace(beforeScriptCfg["source"].(string), "{{serverURL}}", serverURL, -1)
```

Comment:
instead adding before_script on each case, we can just check if the before_script key and only replacing when exists